components/headerNav/index.tsx

```diff
@@ -243,11 +94,9 @@ const HeaderNav = () => {
           {/*    <Anchor>The CIMUN Chronicle</Anchor> */}
           {/*  </BurgerListItem> */}
           {/* </Link> */}
-          <Link href="/leahy" legacyBehavior>
-            <BurgerListItem>
-              <Anchor>Leahy Fellowship</Anchor>
-            </BurgerListItem>
-          </Link>
+          <BurgerListItem>
+            <Anchor href='https://forms.gle/pRseknBXa8EzTmuD9' target="_blank">Leahy Fellowship <FontAwesomeIcon icon='arrow-up-right-from-square' /></Anchor>
```

Comment:
Even though this is correct, it's aesthetically jarring to me and I would rather us work towards creating a whole page so that we don't need to have this icon on every page in the headerNav

Reply:
agreed, icon removed